oauth-flow/docker-compose.yml

```diff
@@ -1,34 +1,38 @@
 version: "3.7"
 services:
   nodeA-backend:
```

Comment:
I think it is confusing that nodeA is an nginx container while nodeB is the nuts-node where nodeA-backend it the nuts-node for A.

Reply:
Might be, but out of scope for this PR